src/sentry/search/events/fields.py

```diff
@@ -1162,6 +1248,103 @@ def is_accessible(self, acl=None):
+                                    #                 1,
+                                    #             ],
+                                    #         ],
+                                    #         "'fcp'",
+                                    #     ],
+                                    # ],
+                                    # "measurements[fcp]",
+                                    "duration",
+                                ],
+                            ],
+                            [
+                                "multiply",
+                                [
+                                    [
+                                        "tupleElement",
+                                        [
+                                            ArgValue("threshold_config"),
+                                            2,
+                                        ],
+                                    ],
+                                    4,
+                                ],
+                            ],
+                        ],
+                    ],
+                ],
+                None,
+            ],
+            default_result_type="number",
+        ),
+        Function(
+            "user_misery_new",
+            # To correct for sensitivity to low counts, User Misery is modeled as a Beta Distribution Function.
+            # With prior expectations, we have picked the expected mean user misery to be 0.05 and variance
+            # to be 0.0004. This allows us to calculate the alpha (5.8875) and beta (111.8625) parameters,
+            # with the user misery being adjusted for each fast/slow unique transaction. See:
+            # https://stats.stackexchange.com/questions/47771/what-is-the-intuition-behind-beta-distribution
+            # for an intuitive explanation of the Beta Distribution Function.
+            required_args=[FunctionAliasArg("threshold_config")],
+            optional_args=[
+                with_default(5.8875, NumberRange("alpha", 0, None)),
+                with_default(111.8625, NumberRange("beta", 0, None)),
+            ],
+            calculated_args=[
+                {"name": "parameter_sum", "fn": lambda args: args["alpha"] + args["beta"]},
+            ],
+            transform=(
+                "ifNull(divide(plus(uniqIf(user, greater("
+                "multiIf("
+                "equals(tupleElement({threshold_config}, 1), 'lcp'),"
+                "if(has(measurements.key, 'lcp'), arrayElement(measurements.value, indexOf(measurements.key, 'lcp')), NULL),"
+                "equals(tupleElement({threshold_config}, 1), 'fcp'),"
+                "if(has(measurements.key, 'fcp'), arrayElement(measurements.value, indexOf(measurements.key, 'fcp')), NULL),"
+                "duration"
+                "), "
+                "multiply(tupleElement({threshold_config}, 2), 4)"
+                ")), {alpha}), plus(uniq(user), {parameter_sum})), 0)"
```

Comment:
Not a fan that this has to hard code the expression for lcp within sentry when it should only live in snuba. Though this is something we need `SnQL` to address. 😞

Reply:
👀 wow this is a beast, can we do a `"""` multiline string so we can indent etc and get some readability? short-term character waste is fine since we'll be replacing this when we migrate to `SnQL`